aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs

```diff
@@ -51,19 +51,19 @@ public class CanvasManager : MonoBehaviour
     private string _currentSceneName = "";
 
     /// <summary>
-    /// To ensure the game settings and menu UI aren't on at the same time.
+    /// Prevent the settings toggle when the variable is true.
     /// </summary>
-    private bool _isOnGameSettings;
+    private bool _preventSettingsToggle;
 
     /// <summary>
-    /// To ensure the menu settings and menu UI aren't on at the same time.
+    /// To ensure the game settings and menu UI aren't on at the same time.
     /// </summary>
-    private bool _isOnMenuSettings;
+    public bool IsOnGameSettings { get; private set; }
```

Comment:
waarom public gemaakt? Had je die nodig?

Reply:
Ja in GridPlacerEditor.cs